sync/landing.py

```diff
@@ -638,7 +638,9 @@ def next_try_push(self, retry=False):
                                rebuild_count=0,
                                try_cls=trypush.TryFuzzyCommit,
                                full=True,
-                               exclude=["devedition", "ccov"])
+                               queries=["web-platform-tests !devedition !ccov !fis",
+                                        "web-platform-tests fis !devedition !ccov !asan "
```

Comment:
Maybe I am misunderstanding Mach, but the help text for `mach try fuzzy` says for argument -q: `Specifying multiple times schedules the union of computed tasks.`. Won't this result in a decision task like `web-platform-tests !devedition !ccov !fis web-platform-tests fis !devedition !ccov !asan`, resulting in issues? notably having `!fis` and `fis`.

Reply:
No, it basically runs the queries one after another and adds in all the tasks selected by any query. You can try it locally in a gecko checkout with `./mach try fuzzy --no-push <queries>`.